sdk/resourcemanager/azure-resourcemanager-privatedns/README.md

```diff
@@ -0,0 +1,40 @@
+# Azure Resource Manager Private DNS client library for Java
+
+Azure Resource Manager Private DNS client library for Java
+
+## Getting started
```

Comment:
TODO: it need to refer to common README.md (probably will be placed in sdk/resourcemanager)